### PR TITLE
Remove bundled VNC server

### PR DESCRIPTION
**Pull request by manics**
Fixes https://github.com/jupyterhub/jupyter-remote-desktop-proxy/issues/27 (incorrect licensing, noarch metadata doesn't match bundled executable in pypi package)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ This is based on https://github.com/ryanlovett/nbnovnc.
 
 When this extension is launched it will run a Linux desktop on the Jupyter single-user server, and proxy it to your browser using VNC via Jupyter.
 
-If a `vncserver` executable is found in `PATH` it will be used, otherwise a bundled TightVNC server is run.
-You can use this to install vncserver with support for other features, for example the [`Dockerfile`](./Dockerfile) in this repository installs TurboVNC for improved OpenGL support.
+This extension requires a VNC server to be installed on the system, along with a desktop or equivalent.
+For example, see the [`Dockerfile`](./Dockerfile) in this repository which installs TigerVNC and XFCE4.
```
Comment (yuvipanda) [inline README.md:16]:
```suggestion
For example, see the [`Dockerfile`](./Dockerfile) in this repository which installs [TigerVNC](https://tigervnc.org/) and XFCE4.
```

File: README.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ This is based on https://github.com/ryanlovett/nbnovnc.
 
 When this extension is launched it will run a Linux desktop on the Jupyter single-user server, and proxy it to your browser using VNC via Jupyter.
 
-If a `vncserver` executable is found in `PATH` it will be used, otherwise a bundled TightVNC server is run.
-You can use this to install vncserver with support for other features, for example the [`Dockerfile`](./Dockerfile) in this repository installs TurboVNC for improved OpenGL support.
+This extension requires a VNC server to be installed on the system, along with a desktop or equivalent.
```
Comment (yuvipanda) [inline README.md:15]: Given we have code [like this](https://github.com/jupyterhub/jupyter-remote-desktop-proxy/blob/a0f7620952a32b77de80ba99d6bbdb34d6b27c5b/jupyter_remote_desktop_proxy/setup_websockify.py#L20) I'd say we only support tigervnc and turbovnc right now. We should explicitly mention that, and say that other VNC servers *may* work if they are available on the terminal as a `vncserver` executable.

**Review by yuvipanda — CHANGES_REQUESTED** [review]
w00t! Some documentation changes suggested, but after that I think this can be good to go.

Should also help us get a conda-forge package out.

**Comment by consideRatio** [comment]
Amazing work on things @yuvipanda and @manics!!!

I've not reviewed the changed lines in the PR but I think this is a good decision!

**Comment by yuvipanda** [comment]
@manics I've pushed a change with suggested new wording. If this looks good to you, I think we should merge and make a release!

**Review by consideRatio — COMMENTED** [review]
Nice this looks great!

@yuvipanda I opened https://github.com/jupyterhub/jupyter-remote-desktop-proxy/issues/87 about making the v2.0.0 release.

**Review by consideRatio — APPROVED** [review]
I pushed [2c210fe](https://github.com/jupyterhub/jupyter-remote-desktop-proxy/pull/84/commits/2c210fe966eeeaeb8b7ba3e2827683cdf7d29001) as we now no longer copy files to the share folder. It just includes a single bash script that passed pre-commit checks without change.

**Review by manics — COMMENTED** [review]
Thanks! Feel free to push anything else you want to this PR or merge.

File: README.md [hunk 3]
```diff
@@ -28,19 +36,20 @@ You can use this to install vncserver with support for other features, for examp
...
    dbus-x11
-   libgl1-mesa-glx
    xfce4
    xfce4-panel
    xfce4-session
    xfce4-settings
    xorg
    xubuntu-icon-theme
+   tigervnc-standalone-server
+   tigervnc-xorg-extension
```
[the remark this answers is not in the document]
Comment (consideRatio) [reply README.md:52]: I don't understand the implications to say =/ @yuvipanda ?

File: README.md (hunk 1 above)
Comment (consideRatio) [reply README.md]: I think this was resolved by [94cf5b6](https://github.com/jupyterhub/jupyter-remote-desktop-proxy/pull/84/commits/94cf5b636de529f376e36f473908474d54f8bc05)

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (consideRatio) [reply README.md:52]: Tracked in https://github.com/jupyterhub/jupyter-remote-desktop-proxy/issues/92.

**Pull request merged**